include/trigger/Issues.hpp

```diff
@@ -189,6 +189,14 @@ ERS_DECLARE_ISSUE_BASE(trigger,
                        "Factory could not find requested item " << item << ".",
                        ((std::string)item),
                        ERS_EMPTY)
+
+ERS_DECLARE_ISSUE_BASE(trigger,
+                       ConfigurationProblem,
+                       appfwk::GeneralDAQModuleIssue,
+                       "Configuration error: " << item,
+                       ((std::string)item),
```

Comment:
seems like this is missing the `((std::string)name),` part which passes name of the module to the errors platform (see other issues)